bundles/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting5Message.java

```diff
@@ -178,9 +180,16 @@ public void encodeMessage(byte[] data) throws RFXComException {
         subType = fromByte(SubType.class, super.subType);
 
         sensorId = (data[4] & 0xFF) << 16 | (data[5] & 0xFF) << 8 | (data[6] & 0xFF);
-        unitCode = data[7];
 
-        command = fromByte(Commands.class, data[8], subType);
+        switch (subType) {
+            case LEGRAND_CAD:
+                unitCode = 0x01;
```

Comment:
Is this really needed, reading the SDK unit-codes are not used with a lot the sub-types so this distinction is not perse correct